src/docs/sphinx/quickstart.rst

```diff
@@ -267,28 +267,263 @@ must be specified using either:
 
 ``python3 scripts/uberenv/uberenv.py --spack-config-dir=scripts/spack/configs/linux_ubuntu_18 --prefix=../path/to/install``
 
-Preparing OSX for Serac Installation
+Building Serac dependencies on MacOS
```

Comment:
May I suggest that the reader be warned about the existence of this section back in ``Building Serac's Dependencies via Spack/uberenv`` ?

This quick-start guide did not pass the lazy sequential reader test: as a lazy sequential reader, on my mac I run
``python3 scripts/uberenv/uberenv.py --prefix=../serac_libs``, then it fails, then I read more :)